src/@next/components/atoms/AddNewTile/test.tsx

```diff
@@ -0,0 +1,15 @@
+import { mount } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { AddNewTile } from ".";
+
+describe("<AddNewTile />", () => {
+  // Example test
```

Comment:
Remove comment and make the test name more explicit like "renders correct caption based on provided prop"

Reply:
Fixed 🌮